packages/api-sdk/src/medical/models/common/address.ts

```diff
@@ -1,17 +1,13 @@
-import { stripNonNumericChars } from "@metriport/shared";
+import {
+  stripNonNumericChars,
+  geoCoordinateSchema,
+  usStateForAddressSchema,
+} from "@metriport/shared";
 import { z } from "zod";
 import { defaultOptionalString, defaultString } from "../../../shared";
-import { usStateSchema, usTerritorySchema } from "./us-data";
 
 const zipLength = 5;
 
-export const geoCoordinateSchema = z.object({
```

Comment:
I can't find any use for this outside this function... so I no longer export it from api-sdk in this PR.